pl-backend/Program.cs

```diff
@@ -6,6 +10,20 @@
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
+builder.Services.AddScoped<ITokenService, TokenService>();
+builder.Services.AddScoped<IUserService, UserService>();
+
```

Comment:
It's something Szymon forgot to add, but please you do add it, since its crucial to make context work
```suggestion
builder.Services.AddDbContext<DataContext>();

```